Flick/Cells/CollaboratorTableViewCell.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  CollaboratorTableViewCell.swift
+//  Flick
+//
+//  Created by Lucy Xu on 6/21/20.
+//  Copyright © 2020 flick. All rights reserved.
+//
+
+import UIKit
+
+class CollaboratorTableViewCell: UITableViewCell {
+
+    // MARK: - Private View Vars
+    private let isSelectedIndicatorImageView = UIImageView()
+    private let nameLabel = UILabel()
+    private let ownerLabel = UILabel()
+    private let selectIndicatorImageView = UIImageView()
+    private let userImageView = UIImageView()
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        selectionStyle = .none
+
+        nameLabel.font = .systemFont(ofSize: 16)
+        nameLabel.textColor = .black
+        addSubview(nameLabel)
+
+        ownerLabel.text = "Owner"
+        ownerLabel.font = .systemFont(ofSize: 16)
+        ownerLabel.textColor = .mediumGray
+
+        userImageView.layer.cornerRadius = 20
+        userImageView.layer.backgroundColor = UIColor.darkBlueGray2.cgColor
+        addSubview(userImageView)
+
+        addSubview(selectIndicatorImageView)
```

Comment:
one more note: i thought it might be better to use a UIView for as selectIndicator instead of UIImageView
something like 
``` swift
        let selectView = UIView()
        let selectSize = CGSize(width: 20, height: 20)
        selectView.layer.cornerRadius = selectSize.width / 2
        selectView.layer.borderWidth = 2
        selectView.layer.borderColor = UIColor.lightGray.cgColor
        addSubview(selectView)
```
since it's just a simple circle with border. that way we're relying less on images. 
if it's selected, we can change border color `layer.borderColor = UIColor.gradientPurple.cgColor`, make background color white, and add in the `isSelectedIndicatorImageView`